### ISSUE TITLE
Leading comments in base64 encoded irules stripped by as3

### ISSUE THREAD

**Issue by kwinkelman-bw** [F5Networks/f5-appsvcs-extension#252]
### Environment
 * Application Services Version: 3.18.0
 * BIG-IP Version: BIG-IP 13.1.3.2 Build 0.0.4 Point Release 2 

### Summary
When applying iRules that have the first line as a comment, the leading comment is stripped from the "applied" iRule. This leads to a non-idempotent state where every time I apply an as3 config, these rules get deleted and updated, even when the iRule has not changed.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "msg": {
        "$schema": "https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/3.18.0/as3-schema.json",
        "action": "deploy",
        "class": "AS3",
        "declaration": {
            "class": "ADC",
            "id": "tenant.dev",
            "tenant": {
                "Shared": {
                    "template": "shared",
                    "test_irule": {
                        "class": "iRule",
                        "iRule": {
                            "base64": "IyB0aGlzIGlzIGEgdGVzdCBjb21tZW50CndoZW4gSFRUUF9SRVFVRVNUIHsKICAgICAgICBIVFRQOjpoZWFkZXIgcmVwbGFjZSBYLVJlYWxtIHN0YW5kYXJkCiAgICB9"
                        }
                    }
                }
            }
        }
    }
}
```

2. Observe the following error response:
The response is not an error - as3 accepts a valid declaration.

### Expected Behavior
The base64 decode of the above text is: ```# this is a test comment
when HTTP_REQUEST {
        HTTP::header replace X-Realm standard
    }```

I expect that entire contents to be found in the declared iRule.

### Actual Behavior
When that declaration is applied, what I see in the irule is:
``` # list ltm rule test_irule
ltm rule test_irule {
    partition tenant
when HTTP_REQUEST {
        HTTP::header replace X-Realm standard
    }
}
```

**Comment by dstokesf5** [comment]
Thank you for reporting this. I have confirmed the bug and added it to our internal product backlog as AUTOTOOL-1513.

**Comment by kwinkelman-bw** [comment]
I also determined this was happening to trailing comments as well.

**Comment by j-zehnder** [comment]
Is it possible to get an updated timeline on the expected fix for this issue?

**Comment by dstokesf5** [comment]
This has been resolved in the 3.33.0 release.
